### ISSUE TITLE
Error during upgrade from 3.21.1 to 3.22.1

### ISSUE THREAD

**Issue by Laul0** [docker-library/ghost#228]
After performing a Docker Pull

```bash
docker-compose -f ghost-compose.yml pull
```

And a up

```bash
docker-compose -f ghost-compose.yml up -d
```

The following error is displayed and Ghost does not up:

```
[2020-07-02 22:32:39] INFO Adding force_i18n, permalinks, and members_session_secret to "settings" table.,
    at process.topLevelDomainCallback (domain.js:137:15),
Empty .update() call detected! Update data does not contain any values to update. This will result in a faulty query. Table: settings. Columns: value.,
[2020-07-02 22:32:39] ERROR Empty .update() call detected! Update data does not contain any values to update. This will result in a faulty query. Table: settings. Columns: value.,
{"config":{"transaction":true},"name":"05-migrate-members-subscription-settings.js"},
Error ID:,
"Error occurred while executing the following migration: 05-migrate-members-subscription-settings.js",
    300,
----------------------------------------,
MigrationScriptError: Empty .update() call detected! Update data does not contain any values to update. This will result in a faulty query. Table: settings. Columns: value.,
    at /var/lib/ghost/versions/3.22.1/node_modules/knex-migrator/lib/index.js:1055:19,
    at MigrationScriptError.KnexMigrateError (/var/lib/ghost/versions/3.22.1/node_modules/knex-migrator/lib/errors.js:7:26),
    at new MigrationScriptError (/var/lib/ghost/versions/3.22.1/node_modules/knex-migrator/lib/errors.js:25:26),
    at QueryCompiler_MySQL._prepUpdate (/var/lib/ghost/versions/3.22.1/node_modules/knex/lib/query/compiler.js:820:13),
Error: Empty .update() call detected! Update data does not contain any values to update. This will result in a faulty query. Table: settings. Columns: value.,
    at QueryCompiler_MySQL.update (/var/lib/ghost/versions/3.22.1/node_modules/knex/lib/dialects/mysql/query/compiler.js:24:26),
    at QueryCompiler_MySQL.toSQL (/var/lib/ghost/versions/3.22.1/node_modules/knex/lib/query/compiler.js:62:29),
    at Builder.toSQL (/var/lib/ghost/versions/3.22.1/node_modules/knex/lib/query/builder.js:72:44),
    at /var/lib/ghost/versions/3.22.1/node_modules/knex/lib/runner.js:31:36,
    at /var/lib/ghost/versions/3.22.1/node_modules/knex/lib/runner.js:260:24,
    at tryCatcher (/var/lib/ghost/versions/3.22.1/node_modules/bluebird/js/release/util.js:16:23),
    at processImmediate (internal/timers.js:456:21),
    at Async._drainQueues (/var/lib/ghost/versions/3.22.1/node_modules/bluebird/js/release/async.js:102:5),
    at Immediate.Async.drainQueues (/var/lib/ghost/versions/3.22.1/node_modules/bluebird/js/release/async.js:15:14),
    at _drainQueueStep (/var/lib/ghost/versions/3.22.1/node_modules/bluebird/js/release/async.js:93:12),
    at _drainQueue (/var/lib/ghost/versions/3.22.1/node_modules/bluebird/js/release/async.js:86:9),
    at Promise._settlePromises (/var/lib/ghost/versions/3.22.1/node_modules/bluebird/js/release/promise.js:729:18),
    at Promise._settlePromise0 (/var/lib/ghost/versions/3.22.1/node_modules/bluebird/js/release/promise.js:649:10),
    at Promise._settlePromise (/var/lib/ghost/versions/3.22.1/node_modules/bluebird/js/release/promise.js:604:18),
    at Promise._settlePromiseFromHandler (/var/lib/ghost/versions/3.22.1/node_modules/bluebird/js/release/promise.js:547:31)
```

**Comment by Laul0** [comment]
After a rollback to 3.21.1, I lost all code injection (header and footer) 😭

**Comment by yosifkit** [comment]
https://github.com/docker-library/ghost/issues/227#issuecomment-653119373

**Comment by yosifkit** [comment]
Fixed in `3.22.2` "Hardened members subscription migration against missing data (https://github.com/TryGhost/Ghost/pull/12009)"
